data/World/Shadow Temple MQ.json

```diff
@@ -24,7 +30,15 @@
         "dungeon": "Shadow Temple",
         "locations": {
             "Shadow Temple MQ Compass Chest": "True",
-            "Shadow Temple MQ Hover Boots Chest": "can_play(Song_of_Time) and Bow"
+            "Shadow Temple MQ Hover Boots Chest": "can_play(Song_of_Time) and Bow",
+            "Shadow Temple MQ Whispering Walls Pot 1": "True",
+            "Shadow Temple MQ Whispering Walls Pot 2": "True",
+            "Shadow Temple MQ Compass Room Pot 1": "True",
+            "Shadow Temple MQ Compass Room Pot 2": "True",
+            "Shadow Temple MQ Whispering Walls Flying Pot 1": "can_play(Song_of_Time)",
+            "Shadow Temple MQ Whispering Walls Flying Pot 2": "True",
+            "Shadow Temple MQ Whispering Walls Flying Pot 3": "can_play(Song_of_Time)",
+            "Shadow Temple MQ Whispering Walls Flying Pot 4": "True"
```

Comment:
Nitpick: These two sets of flying pots with different requirements should have different names.